cpp/include/cudf/quantiles.hpp

```diff
@@ -42,5 +42,32 @@ quantile(column_view const& input,
          interpolation interp = interpolation::LINEAR,
          order_info column_order = {});
 
+/* @brief Gathers the rows corrosponding to the desired quantiles. The indices
```

Comment:
#KeepBriefsBrief :) 
```suggestion
/* @brief Gathers the rows corrosponding to the desired quantiles. 
*
* The indices
```

Maybe since there was some question/confusion about if 0.0 and 1.0 always refer to the first/last row, we should document it.